### PR TITLE
Feature/22 7 search input title heading

### PR DESCRIPTION
**Pull request by nhi-ngo**
-add title heading: There are few I can think of: Be Safe Where You Are, Connect And Stay Safe, etc... I go with the first one for now until we come up with a better phrase.
-fetch list of states upon page load
-add .eslintrc for linting

### CODE REVIEW

**Comment by theborgh** [comment]
Looks good! I just don't understand why the first select option ("Select a state") is hidden :) but ok to merge from me

**Review by theborgh — APPROVED** [review]

File: src/components/Navbar/SearchLocation/SearchLocation.js [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+  function fetchListOfStates() {
+    setStates(data);
   }
+
+  return (
+    <div>
+      <select id="states">
+        <option value="" hidden>
+          Select a state
+        </option>
```
Comment (theborgh) [inline src/components/Navbar/SearchLocation/SearchLocation.js:24]: Referring to this in my comment above

**Comment by cechae** [comment]
Looks good!!

File: src/components/Navbar/SearchLocation/SearchLocation.js (hunk 1 above)
Comment (nhi-ngo) [reply src/components/Navbar/SearchLocation/SearchLocation.js:24]: When you click on the dropdown, "select a state" won't show as an option :)

**Review by cechae — APPROVED** [review]

**Pull request merged**